index.js

```diff
@@ -1,88 +1,129 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+let counter = 1
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+console.log("%cQuestion 1", "color: red") 
+counter = 2
+console.log(counter) 
+// => 2
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
-// *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
+// // *** otherwise, the error will stop the rest of your code from running
+const name = 'James'
+console.log("%cQuestion 2", "color: red") 
 
-// console.log(name)
-// => "Raffy"
+console.log(name)
+// // => "Raffy"
 // name = "Not Raffy" 
-// => TypeError
-// console.log("%c----------", "color: red") 
+// // => TypeError
+console.log("%c----------", "color: red") 
 
 
 
-// ***** Question 3 *****
+// // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red") 
 
-// console.log(drinkWater(12))
-// // => "Man I sure am thirsty"
-// // => "Ahh that hits the spot"
-// // => 11
-// console.log("%c----------", "color: red") 
+function drinkWater(current_thirst_level){
+      console.log("Man I sure am thirsty")
+      current_thirst_level -= 1;
+      console.log("Ahh that hits the spot")
+      return current_thirst_level;
+}
+console.log(drinkWater(12))
```

Comment:
Well done. For Javascript, it's standard to use [camel case](https://upload.wikimedia.org/wikipedia/commons/thumb/c/c8/CamelCase_new.svg/1200px-CamelCase_new.svg.png) for function and variable names. So instead of `current_thirst_level`, name it `currentThirstLevel`